CHANGELOG.md

```diff
@@ -5,7 +5,10 @@ Version [2.2dev] - XXXX-XX-XX
 -------------------------------------------------------------------------------
 
 **Changed:**
-
+- Refractored data augmentation to use `Albumentation` library.
+- Cleaned jupyter notebooks and reduced from 13 notebooks to 4.
```

Comment:
should be from 13 notebooks to 5 (instead of 4)

Reply:
Yes you are right, earlier we decided to keep 4 notebooks that's why the number is 4. However, we decided to keep 5, but thanks for pointing out, I have updated in commit (
992804f)

Reply:
Done in commit(992804f)